goqtdrv5/cdrv.h

```diff
@@ -217,7 +222,69 @@ inline QByteArray drvGetByteArray(void *param)
         return QByteArray();
     }
     slice_head *sh = (slice_head*)param;
-    return QByteArray(sh->data,sh->len);
+    return QByteArray(sh->data, sh->len);
+}
+
+inline void drvSetByteArray(void *param, const QByteArray& array)
+{
+    if (param == 0) {
+        return;
+    }
+
+    slice_head *sh = (slice_head*)param;
+    sh->len = array.size();
+    sh->data = array.data();
+}
+
+inline QStringList drvGetStringArray(void *param)
```

Comment:
Sounds good.
BTW, to be more explicit, I meant something like this: 
```go
type utf8_info string

func (d utf8_info) String() string {
	return string(d)
}

//export utf8_info_copy
func utf8_info_copy(p unsafe.Pointer, data unsafe.Pointer, size C.int) {
	s := (*reflect.StringHeader)(p)
	s.Data = uintptr(data)
	s.Len = int(size)
}
```

There might be some problematic parts in cdrv.go, possibly related to #3. I'm hoping to check it thoroughly someday when I have time.